src/components/Modal/Modal.tsx

```diff
@@ -75,6 +75,7 @@ export type ModalHook = {
   closeModal: () => void
 }
 
+/** Alternative to redux, usemodal has to be in the component it's self */
 export const useModal = (): ModalHook => {
```

Comment:
A bit confused by the comment - is this to clarify that when you want to track modal state within the component (as opposed to using state management like Redux) you must import the `useModal` hook? Maybe this comment would be better suited in the `example` app code since that is where you can see the difference in implmentation.

Reply:
@suzubara thoughts?

Reply:
I find this comment confusing also, I think this is maybe a note you took while we were pairing? I don't think it is necessary to keep it here